checkov/terraform/checks/resource/aws/VPCDefaultNetwork.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+class VPCDefaultNetwork(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure no default VPC is planned to be provisioned"
+        id = "CKV_AWS_114"
```

Comment:
@ebrucucen since we merged #1023 this ID is taken, please merge from master and take the next ID, I believe it is 148